### DISCUSSION TITLE
Statistics data has changed?

### DISCUSSION THREAD

**Discussion by hazardtrip** [artisan-roaster-scope/artisan#1108] [General]
Hi, hope someone here can help...

The statistics data at the bottom of my saved PDF/Artisan screen has changed from -
"10/01/2023, 11:49 Honduras Honduras - La F... 8334g -17.3%"
to
"MET=240.1C RoR=10.3C/min AUC=704C*min FC=2:36min"

This changed a couple of months back and I have no idea how to switch it back.  If I uncheck the statistics check box I end up with nothing there.
I'm running Artisan 2.6.0 (7088095)

Thanks,

Chris
River City Coffee Roasters

**Comment by roasterdave** [comment]
Hi Chris,

A right click on the statistics display at the bottom will toggle between the two displays of data.

Now it's time to upgrade to the latest version [v2.8.2](https://github.com/artisan-roaster-scope/artisan/releases/latest).

**Reply by hazardtrip** [reply]
Ahhhhh, that sorted it. Thanks heaps.  Yes was refusing to upgrade until I sorted this out = stubborn :-)
